DDG4/python/DDSim/Helper/ParticleHandler.py

```diff
@@ -153,4 +153,15 @@ def setupUserParticleHandler(self, part, kernel, DDG4):
 
       part.adopt(user)
 
+    elif self.userParticleHandler == "Geant4TVUserParticleHandler":
+      user = DDG4.Action(kernel, "%s/UserParticleHandler" % self.userParticleHandler)
+      try:
+        user.TrackingVolume = DDG4.Kernel().detectorDescription().trackingVolume()
```

Comment:
```suggestion
        user.TrackingVolume = kernel.detectorDescription().trackingVolume()
```
But this assignment gives
```
DDSim.Helper.ParticleHandler ERROR tracking_volume missing in detector model. KeyError('Geant4Action::SetProperty [Unhandled]: Cannot set UserParticleHandler.TrackingVolume = <cppyy.gbl.dd4hep.Volume object at 0x3905b9f0>')
```

Reply:
But then I had removed the property before....